src/flags.cpp

```diff
@@ -0,0 +1,38 @@
+#include "../include/flags.h"
```

Comment:
Your makefile defines `include/` as an include dir, so having `#include "include/flags.h"` will make the compiler look inside `include/include/flags.h`, which doesn't exist. At least I think that's the problem.